circle.yml

```diff
@@ -17,9 +17,12 @@ machine:
 dependencies:
   cache_directories:
     - "~/cache"
+    - "~/.npm"
   override:
     - sudo apt-get -q update; sudo apt-get install -q -y libgif-dev libsasl2-dev
     - ./scripts/install_cmake.sh
+    # The real npm cache is in ~/.npm, but CircleCI tries to cache node_modules, breaking upgrades
+    - rm -rf ./node_modules
```

Comment:
One would think, but my development environment is always fine when I switch onto a branch with a different `package.json` / `package-lock.json` and rebuild in-place.

All this will go away in CircleCI 2, when we have much more direct control over what is cached.